src/api/Jalasoft.TeamUp.Resumes.API/local.settings.json

```diff
@@ -4,6 +4,8 @@
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
     "SQLConnetionString": "Server=tcp:sql-tuapi-dev-bra.database.windows.net,1433;Initial Catalog=sqldb-tuapi-dev-bra;Persist Security Info=False;User ID=n1d0d3v;Password=D3vL3v3l3;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
+    
+      // "SQLConnetionString": "Server = DESKTOP-33OMQKP\\SQLEXPRESS; DataBase = TeamUpResumes; User ID = AppConnection; Password = 12345678;"
```

Comment:
You need remove this line, the comments in the deploy project is´t a good practice

Reply:
the pipeline is not working

Reply:
Done